styles/components/dropdown.scss

```diff
@@ -88,15 +102,11 @@
     border-top-color: $brand-white;
   }
 
-  // Right
-  &.-right > .menu::before {
-    left: initial;
-    right: $dropdown-arrow-size;
-  }
-
-  &.-right > .menu::before,
-  &.-right > .menu::after {
-    left: initial;
-    right: $dropdown-arrow-size;
+  //Up Right
+  &.-up &.-right> .menu::before,
+  &.-up &.-right> .menu::after {
```

Comment:
I think this modifier is a bit too much. We don't need to combine modifiers like this. This should suffice:
```sass
.dropdown {
  &.-up {
  }

  &.-right {
  }
}
```
Unless there's an issue? .. :confused: